### DISCUSSION TITLE
Using OpenSea SDK to list and sell Polygon NFTs

### DISCUSSION THREAD

**Discussion by nuCodeIO** [ProjectOpenSea/seaport#587] [General]
I have been trying out https://github.com/ProjectOpenSea/opensea-js
However, it only supports Ethereum Mainnet Rinkeby.

Is it possible to use this for Polygon?

Or is there another SDK/library that supports listing and selling Polygon NFTs on OpenSea?

Thanks!

**Comment by 0xAskar** [comment]
It seems like in the Discord, they are working on it but there is no ETA at the moment to support it for 2.0.
<img width="648" alt="image" src="https://user-images.githubusercontent.com/37524379/180871876-13585986-30b5-4d62-a2a6-1e8377e3e957.png">
